examples/export-avatars-positions.js

```diff
@@ -0,0 +1,147 @@
+
+//Created by Teodoro Dannemann --->  https://github.com/teo523    
+//Credits to Zach Capalbo ---> https://github.com/zach-capalbo/hubs-client-bot
+
+//This example uses hubs-bot for extracting (x,y,z) positions of every avatar connected to the room.
+//You can combine this with sockets.io to stream data to clients (see the following link 
+// where I used (x,y) position to create a live map of the room) 
+// ----> https://github.com/teo523/hubs-client-bot/tree/metapoiesis
+
+const {HubsBot} = require('../index.js')
+const fs = require('fs')
+
+
+class ImportExportBot extends HubsBot {
```

Comment:
Might be clearer to have a different class name here, so as not to confuse readers with the other example

```suggestion
class ExportAvatarPositionsBot extends HubsBot {
```